### DISCUSSION TITLE
Gene order in SCT scale.data slot

### DISCUSSION THREAD

**Discussion by K10308** [satijalab/seurat#4377] [Q&A] [answered]
Hi, I am wondering how the genes are ordered in each data matrix (9 in total, in assay "RNA", "SCT", "integrated", each has 3 slots including counts, data, scale.data). Are they ordered by variance for those data slots where the row dimension equal to the number of variable features? I also notice that even though the genes in SCT scale.slot are the same with integrated scale.data slot, the order has changed. Any reason for that? 

Many thanks!

**Comment by timoast** [comment, marked as answer]
The order of features in the matrix is arbitrary, they are not sorted by variance or anything like that
